src/checkbox.js

```diff
@@ -0,0 +1,24 @@
+const taskCompleted = (e, items) => {
+  const complete = e.target;
+  if (complete.checked) {
+    complete.parentElement.classList.add('completed');
+    for (let i = 0; i < items.length; i += 1) {
+      // eslint-disable-next-line eqeqeq
```

Comment:
- [ ] OPTIONAL
 You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.